### ISSUE TITLE
[Help] Host or domain name not found. Name service error for name

### ISSUE THREAD

**Issue by darmawan01** [panubo/docker-postfix#37]
Got this issues:

```
postfix/error[3180]: DF27F16456B: to=<omama@yahoo.co.id>, relay=none, delay=369825, delays=369825/0.15/0/0, dsn=4.4.3, status=deferred (delivery temporarily suspended: Host or domain name not found. Name service error for name=yahoo.co.id type=MX: Host not found, try again)
```

Here is my docker compose:

```
image: panubo/postfix
    environment:
      MAILNAME: domain_name.id 
      MYNETWORKS: "127.0.0.1/32, 10.21.0.0/24"
      USE_TLS: 'no'
    volumes:
      - /etc/resolv.conf:/etc/resolv.conf
      - ./mail/spool/postfix:/var/spool/postfix
    restart: always
    ports:
      - "587:25"
```

Anyone can help, what i miss in config ?

**Comment by darmawan01** [comment]
more logs now:

```
Dec  4 14:44:07 postfix/smtp[2097]: E89BD1644BA: to=<omama@gmail.com>, relay=none, delay=22988, delays=22838/0.09/150/0, dsn=4.4.1, status=deferred (connect to alt4.gmail-smtp-in.l.google.com[209.85.147.27]:25: Connection timed out)


```

**Comment by darmawan01** [comment]
i see, looks like my outbound 25 port is blocked by provider. closing this issues
